utils/gitlab_api.py

```diff
@@ -188,6 +188,46 @@ def create_app_interface_reporter_mr(self, reports, email_schema,
         return self.create_mr(branch_name, target_branch,
                               commit_message, labels=labels)
 
+    def create_app_interface_notificator_mr(
+        self, notification,
+        notification_path="/app-interface",
+        email_schema="/app-interface/app-interface-email-1.yml"
+    ):
+        # labels = ['automerge']
+        labels = ['do-not-merge/hold']
+        prefix = 'app-interface-notificator'
+        target_branch = 'master'
+
+        now = datetime.now()
+        ts = now.strftime("%Y%m%d%H%M%S")
+        isodate = now.isoformat()
+        short_date = now.strftime('%Y-%m-%d')
+
+        branch_name = 'app-interface-notificator-{}'.format(ts)
+        commit_message = '[{}] notification for {}'.format(
+            prefix, isodate
+        )
+
+        self.create_branch(branch_name, target_branch)
```

Comment:
It's a bit confusing that `create_branch()` takes a `source_branch` argument and for that you provide `target_branch`. I mean, this line could well be written like this: `self.create_branch(new_branch=branch_name, source_branch=target_branch)`. Maybe renaming `target_branch` to `main_branch` or just `branch`?
Anyway, maybe it's just me :)

Reply:
Yeah, I agree! It is a confusing. I renamed target_branch to main_branch.

Reply:
* a bit